host/3.0/buster/amd64/powershell/powershell7-appservice.Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Build the runtime from source
+ARG HOST_VERSION=3.0.14785
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+# Build WebJobs.Script.WebHost from source
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
+    dotnet publish -v q /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 && \
+    rm -rf /azure-functions-host/workers/powershell/6 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+# Install extension bundles
+RUN EXTENSION_BUNDLE_VERSION=1.3.3 && \
```

Comment:
Double check the extension bundle version. Currently they are trying to roll out 2.2.0.

Reply:
Thank you. I wanted to intentionally keep it consistent with all the other Dockerfiles in this repo, so we can upgrade bundles in one shot.